src/CustomIndexing/CustomIndexing.jl

```diff
@@ -0,0 +1,130 @@
+module CustomIndexing
```

Comment:
Maybe `Containers`, i.e., `JuMP.Containers`?

Reply:
Yes, I was thinking we should already think about which name the package would have if we move it to avoid breakage but users will need to add `using PackageName` anyway so we can call it `Containers` here and rename later